### PR TITLE
Added few channels and updated Ken Tv Link

### PR DESCRIPTION
**Pull request by Manipulativemachan**

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -321,8 +321,8 @@ http://kcltv.livebox.co.in/kclhls/live.m3u8
 http://198.144.149.82:8080/NOTV/KCVTV/index.m3u8?token=GTR
 #EXTINF:-1 tvg-id="KCVMovies.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/NY94lnZ.png" group-title="Movies",KCV Movies (576p) [Not 24/7]
 http://198.144.149.82:8080/NOTV/KCVMOVIE/index.m3u8?token=GTR
-#EXTINF:-1 tvg-id="Kentv.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/OnYOCmw.jpg" group-title="",Ken TV (360p) [Not 24/7]
```
Comment (MapGuy11) [inline channels/in.m3u:324]: Keep this URL. We use playlist.m3u8 and this URL works.

File: channels/in.m3u [hunk 2]
```diff
@@ -643,6 +643,8 @@ http://uk4.zecast.com:1935/star-live/suntv.stream/index.m3u8
 https://indo51.gcdn.co/MALYLAM-SuryaMusic/index.m3u8
 #EXTINF:-1 tvg-id="SuryaTV.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/ENrptYg.png" group-title="Entertainment",Surya TV (720p) [Not 24/7]
 https://indo51.gcdn.co/MALYLAM-SuryaHD/index.m3u8
+#EXTINF:-1 tvg-id="Swantham.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://imgur.com/oodBB6x" group-title="Entertainment",Swantham
```
Comment (MapGuy11) [inline channels/in.m3u:646]: http://125.99.241.180:1935/SWANTHAM/live/playlist.m3u8

Please use this

**Review by Manipulativemachan — COMMENTED** [review]
Updated the link as requested

**Comment by Manipulativemachan** [comment]
@Dum4G @LaneSh4d0w  can you fix this and merge it???

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**